pkg/schedule.go

```diff
@@ -0,0 +1,78 @@
+package pkg
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+type Schedule struct {
+	Cmds []ScheduledCommand `json:"commands"`
+}
+
+type ScheduledCommand struct {
+	Cmd      Command `json:"cmd"`
+	Datetime int64   `json:"datetime"`
+}
+
+func strToTime(dtStr string) (time.Time, error) {
```

Comment:
Little confused about how times are coming in and converted, so we are going to have timestamps in the OnTimes[] and those are used to generate new timestamps using the hour and minute etc? How does the logic know to generate a schedule for a specific day?

Reply:
That's a good question, I wasn't really thinking about this. We should probably remove date from this and just use time instead. I also wasn't totally sure of the format for timestamps that will be returned by hydrangea, I'll need to double check that